docs/run-testing-framework.md

```diff
@@ -213,6 +213,44 @@ Don't forget to clean up your resources:
 terraform destroy -auto-approve
 ````
 
+### 2.7 Run batch tests
+
+Batch testing allows a set of tests to be ran synchronously. To do this,
+a `test-case-batch` file is required in the `./terraform` directory. 
+The format of the `test-case-batch` file is as such. The `test-case-batch` 
+file can contain an infinite amount of test cases as
```

Comment:
Infinite?  Surely it's at least bounded by available disk space.

Reply:
Updated